.github/workflows/config/yamllint.yaml

```diff
@@ -0,0 +1,10 @@
+extends: default
+
+rules:
+  # 80 chars should be enough, but don't fail if a line is longer
+  line-length:
+    max: 80
+    level: warning
+
+  indentation:
+    indent-sequences: whatever
```

Comment:
What does it mean by 'whatever' ?

Reply:
Do you think it's a good idea we insist on one of them for sake of consistency ?

Reply:
Nah there's no set guideline one way or the other, especially in k8s sometimes you'll see both in the same file. and sometimes text editors will auto-indent them a certain way which will just cause headaches. so it's okay to let it go, it will still check that the lists are valid